src/filters/SelectSorting/SelectSorting.stories.mdx

```diff
@@ -0,0 +1,75 @@
+import { action } from "@storybook/addon-actions";
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
+import { array, select, withKnobs } from "@storybook/addon-knobs";
+
+import SelectSorting from "./SelectSorting";
+
+<Meta title="Filters|SelectSorting" component={SelectSorting} />
+
+# Select Sorting
+
+### Display radio checkboxes to select sorting options
+
+The width of SelectSorting takes on the width of its container.
+
+<Props of={SelectSorting} />
+
+#### Sort Order is disabled because Sort By doesn't have a selected value.
+
+<Story name="Disabled Sort Order">
+  <SelectSorting
+    state={{
+      by: "",
+      order: "",
+    }}
+    update={action("update-sorting-state")}
+    sortByOptions={["name", "date", "value"]}
+  />
+</Story>
+
+#### Sort Order is enabled because Sort By has a selected value.
+
+<Story name="Enabled Sort Order">
+  <SelectSorting
+    state={{
+      by: "name",
+      order: "",
+    }}
+    update={action("update-sorting-state")}
+    sortByOptions={["name", "date", "value"]}
+  />
+</Story>
+
+### Playground
+
+#### Experiment with these stories with Knobs addon in Canvas mode.
+
+Change the values of the sort state to change what sort options are selected.
+
+<Story name="Sort State Playground" parameters={{ decorators: [withKnobs] }}>
+  <SelectSorting
+    state={{
+      by: select("Sort By", {
+        "Committee Name": "name",
+        "Event Date": "date",
+        "Search Relevance": "value",
+      }),
+      order: select("Sort Order", { Ascending: "asc", Descending: "desc" }),
+    }}
+    update={action("update-sorting-state")}
+    sortByOptions={["name", "date", "value"]}
+  />
+</Story>
```

Comment:
If I understand how all these files play together, these options are not baked into the component themselves but are passed in as parameters? Iirc, we potentially wanted to remove certain sorting options as they were confusing to users. Will have to double back with UX to clarify what those options were.

Reply:
If you are talking about lines 51-55, those strings are just for the knobs tab. 
If you look at the knobs tab [story](https://tohuynh.github.io/cdp-instance/?path=/story/filters-selectsorting--sort-state-playground). The `select` is for the user to interact with the component (to select different options).

sortByOptions prop is definitely passed in as parameters.